swift-sdk/Internal/IterableEmbeddedManager.swift

```diff
@@ -43,6 +43,13 @@ class IterableEmbeddedManager: NSObject, IterableEmbeddedManagerProtocol {
     public func syncMessages(completion: @escaping () -> Void) {
         retrieveEmbeddedMessages(completion: completion)
     }
+    
+    public func handleEmbeddedClick(action: IterableAction) {
+        
+        for listener in listeners.allObjects {
+            listener.onTapAction(action: action)
+        }
+    }
```

Comment:
@evantk91 do we need to track a click event here?

Reply:
No right now that is handled app side

Reply:
Moving it into the handler would make implementation easier. I will do that.